src/main/scala/devices/uart/UARTCtrlRegs.scala

```diff
@@ -12,4 +12,6 @@ object UARTCtrlRegs {
   val ie     = 0x10
   val ip     = 0x14
   val div    = 0x18
+
+  val txdone = 0x20
```

Comment:
Would suggest naming this register with a broader name, such as "txstatus" or "txstat".  That way, if we want to add more TX-related read-only bitfields in the future, we can pack them into the existing register.

Reply:
👍

Reply:
Makes sense.